docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.9"
+
+services:
+  postgresqldb:
+    image: postgres:latest
+    container_name: postgres
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_DB=expense_tracker_db
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=admin
```

Comment:
we need another way of providing the credentials instead of hardcoding in the git repository

Reply:
It is only for local usages instead of using a real database.